From 2c305b5c40b218618bd30ca1ead96865d31a1056 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Sun, 20 Jun 2021 23:29:04 +0300
Subject: [PATCH] [SW-0] gaudi: security: (re)use spi base define

Reuse the available define for HL_SPI_BASE instead of the hard coded
value.

Change-Id: Ifb5cba8f74447ed29afdd0da3e247cfbeb1ae1e4
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/gaudi/security.c b/gaudi/security.c
index bfac2e7..f4e7cb0 100644
--- a/gaudi/security.c
+++ b/gaudi/security.c
@@ -1537,7 +1537,7 @@
 	preboot_secure_hbw_rr(HBW_RR_STM, 0x7FF4000000, 0x3FFFFC000000);
 
 	/* SPI Flash RR 0x7F_F800_0000 - 0x7F_FA00_0000 (32MB) */
-	preboot_secure_hbw_rr(HBW_RR_SPI_FLASH, 0x7FF8000000, 0x3FFFFE000000);
+	preboot_secure_hbw_rr(HBW_RR_SPI_FLASH, HL_SPI_BASE, 0x3FFFFE000000);
 
 	/* Scratchpad SRAM RR 0x7F_FBFE_0000 - 0x7F_FBFF_0000 (64KB) */
 	preboot_secure_hbw_rr(HBW_RR_SP_SRAM, 0x7FFBFE0000, 0x3FFFFFFF0000);
